ansible/inventory

```diff
@@ -14,3 +14,6 @@ switch5
 
 [ptf]
 ptf-1 ansible_host=10.0.0.200 ansible_ssh_user=root ansible_ssh_pass=password
+
+[tacacs_server]
+tacacs_server ansible_host=10.0.0.9 ansible_ssh_user=root ansible_ssh_pass=root
```

Comment:
@liuqu  are you using the default ptf docker within the testbed? I didn't closely follow up PRs in sonic-buildimage, is this server been included and deployed with testbed ptf docker? 
did you add your Tacacs+ design/testing document to sonic/wiki so the community member can follow? (I remembered last time it's from your own repo)

Reply:
Sorry too late to reply. You can find the the test plan in [TACACS+ Test Plan.md](https://github.com/Azure/SONiC/blob/gh-pages/doc/TACACS%2B%20Test%20Plan.md)
TACACS+ server is a host which installed tac_plus, not included in ptf docker. This configuration is used to save the username and password to ensure that TACACS+ server can be logined to execute sudo command.

Reply:
The feature source address for TACACS+ is needed for source address test. It will be pulled request after the current PRs are merged.